simple_use_case/snakemake/Snakefile

```diff
@@ -1,65 +1,86 @@
+configfile: "./config.yaml"
+
+
+# NOTE 
+# $ snakemake --use-conda --cores 1 --config domain_size=4.0
+# does not re-execute the job generated from rule 'generate_mesh'
+# if it was executed before with e.g. the default value 2.0
+# It seems one has to force the execution of 'somerule' in this case
+# $ snakemake -R somerule
 rule generate_mesh:
-	input:
-		"../source/unit_square.geo"
-	output:
-		"preprocessing/unit_square.msh"
-	conda:
-		"../source/envs/preprocessing.yaml"
-	shell:
-		"gmsh -2 -order 1 -format msh2 {input} -o {output}"
+    input:
+        "../source/unit_square.geo"
+    output:
+        msh="preprocessing/square.msh",
+        stdout="preprocessing/gmsh_stdout.txt"
```

Comment:
is stdout of the mesh step still required?

Reply:
my bad, I just forgot to remove it. Thanks!